full_storage_utilization_test.py

```diff
@@ -0,0 +1,200 @@
+import time
+from sdcm.tester import ClusterTester
+from sdcm.utils.tablets.common import wait_for_tablets_balanced
+
+
+class FullStorageUtilizationTest(ClusterTester):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.sleep_time_before_autoscale = 120
+        self.sleep_time_fill_disk = 1800
+        self.soft_limit = self.params.get('diskusage_softlimit') if self.params.get('diskusage_softlimit') is not None else 70
```

Comment:
doesnt `get` allow default value?

Reply:
`SctConfiguration.get` doesn't, on purpose, to make sure all default are listed in specific designated places under `defaults/test_defaults.yaml` or per backend.

and not scattered across the code, in multiple places the same config options might be use.
at some point it was like that, and it was mass changing the default, or even knowing what they are.

I would recommend default to be kept in configuration files, and not in the code like the above.

Reply:
ok, update the code and added these two limits into `defaults/test_defaults.yaml`